src/lib.rs

```diff
@@ -305,7 +305,7 @@
 
 #[doc(inline)]
 pub use self::{
-    bson::{Array, Binary, Bson, DbPointer, Document, JavaScriptCodeWithScope, Regex, Timestamp},
+    bson::{Array, Binary, Bson, DbPointer, Document, JavaScriptCodeWithScope, Regex, Timestamp, UuidRepresentation},
```

Comment:
the `use` for `UuidRepresentation` needs to be gated behind the `uuid-0_8` feature flag, otherwise regular builds will fail (e.g. see [here](https://evergreen.mongodb.com/task_log_raw/rust_bson_compile_only__extra_rust_versions~min_compile_only_patch_fb18ed299db7a7bdf75569d5908ab582621667c6_6144bbe12a60ed25d50fdbcf_21_09_17_16_18_03/0?type=T#L169) for the failure in the evg patch).